pkg/vulnsrc/suse-cvrf/suse-cvrf.go

```
@@ -8,14 +8,15 @@
 	"strconv"
 	"strings"
 
+	"github.com/samber/oops"
```

golangci-lint (1.64.8)

File is not properly formatted

(gci)